app/services/katello/applicability/applicable_content_helper.rb

```diff
@@ -122,20 +123,24 @@ def newest_distinct_installed_packages_query
       end
 
       def applicable_differences
-        consumer_ids = content_facet.send(applicable_units).pluck("#{content_unit_class.table_name}.id")
-        content_ids = fetch_content_ids
+        ActiveRecord::Base.connection.uncached do
```

Comment:
@jlsherrill @ianballou the failure we've been seeing is *probably* related to the tests (or tests env) but it may be wise to proceed with this change to ensure that we're never using cached results in this method since it's so critical to errata. I have not seen the test fail with this change (probably 20 jenkins runs).